sys-devel/clang/clang-16.0.0_pre20221217.ebuild

```diff
@@ -273,6 +273,13 @@ multilib_src_configure() {
 
 		-DPython3_EXECUTABLE="${PYTHON}"
 	)
+
+	if use ppc64 && ! use elibc_musl; then
```

Comment:
Do we actually need this condition given we have the profiles masking as appropriate?

Reply:
ppc64 condition may go actually, for cross purposes too.

musl has to stay for now IMO, strictly unsupported, so it;d better be noop even if someone does something nasty.

```c
bool ToolChain::defaultToIEEELongDouble() const {
  return PPC_LINUX_DEFAULT_IEEELONGDOUBLE && getTriple().isOSLinux();
}
```
https://github.com/llvm/llvm-project/blob/d22f050e15cb8c941d488d9674329db320ef1783/clang/lib/Driver/ToolChain.cpp#L141-L143